### DISCUSSION TITLE
FREEDNS.AFRAID.ORG DO NOT WORKING IN BRASIL

### DISCUSSION THREAD

**Discussion by tararanchaves** [NightscoutFoundation/xDrip#3055] [Bug reports]
I NEED OTHER FREE DNS BECAUSE NIGHTSCOUT IN GOOGLE CLOUD DONT WORK. FREEDNS.AFRAID.ORG DO NOT WORKING IN BRASIL. HELP PLEASE
 TARARAN

**Comment by Navid200** [comment]
Please provide more information.
Is there any forum or blog you can provide a link to where others have confirmed that FreeDNS does not work in Brazil?

**Reply by tararanchaves** [reply]
i try since yesterday acess de address and dont enter

![Captura da Web_29-8-2023_191811_freedns afraid org](https://github.com/NightscoutFoundation/xDrip/assets/74116318/d5ef86e8-fb62-4da5-bdb1-7ef882468501)

**Reply by Navid200** [reply]
Can you try a different computer or phone or browser just to make sure it is not a setting or cache?

**Reply by Navid200** [reply]
Also can you try from a different network?  For example, if you are at work, can you try from home?

**Reply by tararanchaves** [reply]
I try in home and celphone, but dont enter...

**Reply by Navid200** [reply]
If FreeDNS is not accessible in Brazil, unfortunately, there isn't anything we can do about that.  
Some individuals have got a private domain.  That's not free.  We can work with the user if they insist to use a private domain to help them bring up their system.

**Reply by tararanchaves** [reply]
Navid200 iam not programmer and my son need use nightscout, the problem is that: without Freedns the celphone send glicemy reading, but the nightscout dont receve. And i dont have acess. How i resolve this problem? You can help me?

**Reply by Navid200** [reply]
Can you search in forums to see if anyone else in Brazil can confirm that FreeDNS is not reachable in Brazil?
Alternatively, you may find out that there are some limitations in Brazil but still possible.
Finding more information is the first thing to do.

I don't speak the language and am not located there.  So, I am not the right person for doing the research.

Google Cloud Nightscout is only one option.  There are other options for running Nightscout.  

If your son uses xDrip as his CGM and if you have an Android phone, you can use xDrip sync to follow your son as an alternative to Nightscout:
https://navid200.github.io/xDrip/docs/xDripSync.html
Although, it is preferred that you sort out Nightscout anyway.

**Reply by tararanchaves** [reply]
Navid200, thanks for your attention i resolve the problem acess. I turn on de VPN Kaspersky and acess Freedns.afraid.org and rebuilt my domains ans subdomains.

**Comment by Navid200** [comment]
Fantastic!
Thanks for reporting this
